--- AutoDiAttributes.Generator.Tests/DIRegistrationGeneratorTests.cs
+++ AutoDiAttributes.Generator.Tests/DIRegistrationGeneratorTests.cs
@@ -1,13 +1,13 @@
+using System.Collections.Immutable;
+using System.Reflection;
+using System.Text;
+using AutoDiAttributes;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.Extensions.DependencyInjection;
 using Moq;
 using Shouldly;
-using AutoDiAttributes;
 using Xunit;
-using System.Collections.Immutable;
-using System.Reflection;
-using System.Text;

 namespace AutoDiAttributes.Generator.Tests;
